pkg/storageincentives/redistribution/redistribution.go

```diff
@@ -17,7 +17,10 @@ import (
 	"github.com/ethersphere/bee/v2/pkg/transaction"
 )
 
-const loggerName = "redistributionContract"
+const (
+	loggerName      = "redistributionContract"
+	BoostTipPercent = 50
```

Comment:
in a future PR, this could be dynamic based on how far the node is from the end of the round phase. wdyt?

Reply:
That is nice idea